bedrock/firefox/templates/firefox/welcome/page4.html

```diff
@@ -0,0 +1,131 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import send_to_device with context %}
+{% from "macros-protocol.html" import hero with context %}
+
+{% add_lang_files "firefox/sendto" "firefox/welcome/page4" %}
+
+{% extends "firefox/welcome/base.html" %}
+
+{# L10n: HTML page title #}
+{% block page_title %}{{ _('Download the Firefox Browser on your Mobile for iOS and Android') }}{% endblock %}
+
+{% block body_class %}{{ super() }} welcome-page4{% endblock %}
+
+{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-2' %}
+{% set _utm_campaign = 'welcome-4-mobile' %}
+
+{# L10n: the strong tag here is to underline Firefox. #}
+{% block content_intro %}
+{% call hero(
+    title=_('Wallet. Keys. Phone. <strong>Firefox.</strong>'),
+    desc=_('Take privacy with you on every device — and leave the data trackers behind.'),
+    class='mzp-t-firefox mzp-t-dark',
+    include_cta=True,
+    heading_level=1,
+    image_url=None,
+    include_highres_image=False,
+    image_alt=''
+  ) %}
+
+<p class="primary-cta">
+  <a class="js-modal-link mzp-c-button mzp-t-product" href="" data-cta-position="primary"
+    rel="external noopener">{{ _('Get the Firefox App') }}</a>
+</p>
+{% endcall %}
+
+<div id="modal" class="mzp-u-modal-content mzp-l-content">
+  <h2 class="modal-logo">Firefox</h2>
+  <h3>{{ _('Get Firefox on your Phone') }}</h3>
+
+  {% if show_send_to_device %}
+    <p>{{ _('Send the download link right to your phone or email.') }}</p>
+    {{ send_to_device(include_title=False, input_label=_('Enter your phone number or email'), class='vertical') }}
+  {% else %}
+    {# L10n: This string is reused from firefox/whatsnew #}
+    <p>{{ _('Download Firefox for your smartphone and tablet.') }}</p>
+    <div class="qr-code-wrapper">
+      <img src="{{ static('img/firefox/welcome/welcome-qr-firefox.png') }}" id="firefox-qr"
```

Comment:
This QR code is pretty large when displayed, might there be a chance it doesn't easily fit above the fold for people on smaller screens?

![image](https://user-images.githubusercontent.com/400117/69043374-d47a3f00-09ea-11ea-8813-41688a5454ae.png)